README.md

```diff
@@ -5,13 +5,104 @@ Group a list of signals into one hash table signal. The output signal will conta
 
 If *one_value* is True, the Signal attributes will be just a single matching value instead of a list of all matching values. If multiple matches, then the last signal processed will be the value used.
 
+If *group_by* is used, a Signal attribute (key) with the default (pre-assigned) name *group* will have the value of *group_by* in the output signal.
+
+**Example:**
```

Comment:
I know we don't have a template for this yet but I think `Examples` would make a good section heading. I also think it should probably go after the existing sections since those provide a nice at-a-glance view of the block for someone who is familiar with its functionality. I'm open to discussion on this though.